content/oci/oci.go

```diff
@@ -168,6 +168,34 @@ func (s *Store) Delete(ctx context.Context, target ocispec.Descriptor) error {
 	return s.storage.Delete(ctx, target)
 }
 
+// DeleteTree deletes the content matching the descriptor as well as its
+// children from the store.
+// Children will be deleted only if they do not any other predecessors than the
+// given root.
+// DeleteTree may fail on certain systems (i.e. NTFS), if there is a process
+// (i.e. an unclosed Reader) using any of root children.
+func (s *Store) DeleteTree(ctx context.Context, root ocispec.Descriptor) error {
+	predecessors, err := s.Predecessors(ctx, root)
+	if err != nil {
+		return fmt.Errorf("getting image predecessors: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("getting predecessors: %w", err)
```
Or maybe something other than "image" I would think